immut/priority_queue/priority_queue.mbt

```diff
@@ -46,6 +46,50 @@ test "from_array" {
   inspect(queue.peek(), content="Some(5)")?
 }
 
+pub fn to_array[T : Compare](self : ImmutablePriorityQueue[T]) -> Array[T] {
+  let arr = Array::new()
```

Comment:
`let arr = []`

Reply:
is it more efficient to use `let arr = []`?

Reply:
> `let arr = []`

is it more efficient to use `let arr = []`?

Reply:
*idiomatic